packages/cyberstorm/src/components/CustomDropDown.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/CustomDropDown.module.css";
+import { Button } from "./Button";
+import { faChevronDown } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import * as DropdownMenu from "@radix-ui/react-dropdown-menu";
+import { SelectItemProps } from "./CustomSelectItem";
+
+type DropDownProps = {
+  colorScheme?: "default" | "defaultDark" | "primary";
+  defaultOpen?: boolean;
+  icon?: ReactNode;
+  content?: ReactNode[];
+  label?: string;
+};
+
+export const CustomDropDown: React.FC<DropDownProps> = (props) => {
+  const { colorScheme, defaultOpen, icon, content, label } = props;
+
+  return (
+    <div className={styles.root}>
+      <DropdownMenu.Root defaultOpen={defaultOpen}>
+        <DropdownMenu.Trigger asChild>
+          <div>
+            <Button label={label} colorScheme={colorScheme} rightIcon={icon} />
```

Comment:
I'm not keen on hardcoding a `Button` as the trigger component as the dropdown can be applied to a variety of elements. Would it be feasible to be able to provide the trigger element independently as a consumer?

Reply:
I think it can be done, but I don't think we should do it right now, Let's write this down for future work. Does the design currently have any dropdowns that don't use button as the trigger?

Reply:
I don't think any of them currently use a button. Looking at the header, it has two instances of a dropdown, one with the link `Developers` and one with the avatar.

Reply:
Also looking at the `Developers` example, the trigger and the drop down list use different color schemes, which isn't supported by the current implementation.

Reply:
Adding the support in the next commit